Description: Remove the mktemp generated directory
 Upstream make file forgets to remove temporary directories
 created using mktmp command. This patch fixes this problem
 .
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: no
Last-Update: 2013-02-03

--- a/Makefile
+++ b/Makefile
@@ -5,9 +5,11 @@ CURDIR := $(shell pwd)
 TMP := $(shell mktemp -d Gargi.XXX)
 
 default:
-	./generate.pe *.sfd
+	./generate.pe Gargi_2.0.sfd
+	rm -rf ${TMP}
 clean:
-	rm -rf *.ttf
+	rm -rf ${TMP}
+	rm -f *.ttf
 sdist:
 	mkdir -p ${TMP}/${ARCHIVE}
 	cp *.sfd COPYING *.txt Makefile generate.pe ${TMP}/${ARCHIVE}
